tokio-util/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
-# 0.3.0 (February 28, 2020)
+# 0.3.0 (March 4, 2020)
 
-Breaking changes:
+### Changed
 
-- Change codec::Encoder trait to take a generic Item parameter (#1746), which allows
-  codec writers to pass references into `Framed` and `FramedWrite` types.
+- Change `Encoder` trait to take a generic `Item` parameter (#1746)
```

Comment:
Why the phrasing change?

Reply:
Fits our changelog style better.

Reply:
I'd prefer the first variant, as I'm not sure _why_ this change was made.

Reply:
I'd really prefer for the original changelog entry to be maintained.